styles.css

```
@@ -0,0 +1,156 @@
+* {
+    box-sizing: border-box;
+}
+
+.u-list-reset {
+    list-style: none;
+    margin-bottom: 0;
+    margin-top: 0;
+    padding-left: 0;
+}
+
+.u-list-inline li {
+    display: inline-block;
+    margin-left: -9px;
+    margin-right: 15px;
+    margin-top: 15px;
+}
+
+/* ////// base elements /////// 
+//////////////////////////////*/
+
+body {
+    background-color: rgb(244, 136, 90);
+    color: black;
+    font-family: 'Libre Baskerville', serif;
+    font-size: 7px;
+    line-height: 10px;
+}
+
+
+article {
+    background-color: rgb(255, 255, 255);
+    border-radius: 10px;
+    height: 196px;
+    margin: 100px 50px;
+    width: 436px;
+}
+
+img {
+    border-bottom-left-radius: 5%;
+    border-top-left-radius: 5%;
+    max-width: 100%;
+    object-fit: cover;
+}
+
+h2 {
+    color: black;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 12px;
+    line-height: 14px;
+    margin-top: 20px;
+}
+
+h3 {
+    color: rgb(142, 167, 66);
+    font-family: 'Montserrat', sans-serif;
+    font-size: 8px;
+    text-decoration-line: none;
+    text-transform: uppercase;
+}
+
+/* icons of bookmark, star, share */
+ul {
+    position: absolute;
+    right: 0;
+    top: 0;
+}
+
+svg {
+    fill: transparent;
+    height: auto;
+    stroke: rgb(244, 136, 90);
+    stroke-width: 3px;
+    transition: all 300ms ease;
+    width: 10px;
+}
+
+svg:hover,
+svg:focus {
+    fill: rgb(244, 136, 90);
+    stroke: rgb(244, 136, 90);
+}
+
+
+/* ////// class styles /////// 
+/////////////////////////////*/
+
+
+.image-wrapper {
+    display: inline-block;
+    height: 196px;
+    overflow: hidden;
+    position: relative;
+    width: 196px;
+}
+
+.image-wrapper::before {
+    content: '';
+    display: block;
+    height: 100%;
+    position: absolute;
+    width: 100%;
+    border-top-left-radius: 5%;
+    border-bottom-left-radius: 5%;
+}
```

So, weird CSS behavior here -- the `::before` pseudoelement is **blocking** the link you have on this image. Because it is absolutely positioned, you can visualize it as a screen that is covering up the link. If you add the following CSS, nothing will change in how things look, but the pseudoelement will no longer interfere with your image link 👍🏻

```suggestion
    pointer-events: none;
}
```

You only need to add this here; you do **not** need to add it below as well.